shaders_js.go

```diff
@@ -0,0 +1,83 @@
+//+build netgo
+
+package engo
+
+import (
+	"github.com/gopherjs/gopherjs/js"
+)
+
+type Shader interface {
+	Initialize(width, height float32)
+	Pre()
+	Draw(texture *js.Object, buffer *js.Object, x, y, rotation float32)
```

Comment:
Have to leave in a few minutes, so I won't be able to help the next few hours. After that, sure thing.